test/pnts_parser_unit_test.js

```diff
@@ -0,0 +1,46 @@
+import assert from 'assert';
+import PntsParser from '../src/Parser/PntsParser';
+
+function bufferFromString(pnts, size) {
+    const buffer = new ArrayBuffer(size);
+    const typed = new Uint8Array(buffer);
+    let next = 0;
+    for (const line of pnts.split('\n')) {
```

Comment:
You could do one loop and remove the `if`: `for (const word of pnts.split(/\s+/))`

Reply:
Done thanks.